packages/data/addon/adapters/elide-facts.ts

```diff
@@ -50,14 +62,51 @@ export default class ElideFacts extends EmberObject implements NaviFactAdapter {
     });
   }
 
+  /**
+   * @param request
+   * @returns graphql query string for a v2 request
+   */
+  private dataQueryFromRequestV2(request: RequestV2): string {
+    const argStrings = [];
+    const { table, columns, sorts, limit, filters } = request;
+    const columnsStr = columns.map(col => queryStrForField(col.field, col.parameters)).join(' ');
+
+    const filterStrings = filters.map(fil => {
+      const { field, parameters, operator, values } = fil;
+      const fieldStr = queryStrForField(field, parameters);
+      const operatorStr = OPERATOR_MAP[operator] || `=${operator}=`;
+      const valuesStr = values.length > 1 ? `(${values.join(',')})` : `${values[0]}`;
```

Comment:
```suggestion
      const valuesStr = values.join(',');
```

Reply:
We need to wrap multiple values in parentheses, but not a single value.
`(val1,val2)` for multiple
`val1` for single

Reply:
Seems like it works with parentheses wrapped around a single value too